css/index.css

```diff
@@ -1,3 +1,294 @@
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body{
+    display:flex;
+}
+
+.Everything{
+    max-width:1000px;
+    margin: 0 auto;
+    width: 100%;
+}
+.main-nav{
+    display: flex;
+    justify-content: space-between;
+    margin-top: 15px; 
+    width: 80%;
+}
+.brand{
+    display:flex;
+    margin-bottom:50px;
+    justify-content: space-between;
+}
+.main-nav a{
+  display: flex;
+  font-size: 14px;
+  color: gray;
+  text-decoration:none;
+}
+.cta{
+    display: flex;
+    justify-content: space-between;
+    margin-top: 130px;
+    margin-bottom: 100px;
+    
+}
+h1{
+    font-size: 100px;
+    text-align: center;
+}
+.cta button{
+margin-left:120px;
+color: black;
+background-color: white;
+padding-top: 10px;
+padding-bottom: 10px;
+padding-right: 40px;
+padding-left: 40px;
+    
+    
+}
+.top-content{
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px black solid;
+    padding-top: 40px;
+    margin-bottom: 50px;
+}
+header nav a:hover {
+  text-decoration: underline;
+}
+.middle-img{
+    margin-bottom: 50px;
+    width: 100%;
+}
+
+.bottom-content{
+    display: flex;  
+    border-bottom: 1px black solid;
+    padding-bottom:40px;
+    margin-bottom: 50px;
+}
+footer  {
+    text-align: center;
+    
+}
+.features p{
+    margin-right:30px;
+}
+.S{
+    padding-right: 20px;
+}
+.P{
+    padding-right: 20px;
+}
+
+@media(max-width:800px){
+    body{
+    margin-right: 20px;
+    margin-left: 20px;
+    }
+    .brand {
+    display:flex;
+    flex-direction: column-reverse; 
+   }
+   .logo-logo{   
+    display:none;
+    
+    }
+    .cta-text{
+    margin: 0 auto;
+    }
+    footer {
+       margin-bottom: 20px;
+       text-align: center;
+       
+     }
+    .logo{
+    margin: 0 auto;
+    }
+    .middle-img{
+     width: 100%;
+    }
+    .main-nav{
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+    }
+    
+   }
+@media(max-width:500px){
+    body{
+        margin-left:0px;
+        margin-right:0px;
+    }
+    .main-nav {
+    display: flex;
+    flex-direction: column;
+    width:100%; 
+    
+    }
+    
+    .main-nav a{
+        border: 1px solid black; 
+        padding-bottom: 15px;
+        font-size: 20px; 
+        padding-top: 15px;
+    }
+    
+    body{
```

Comment:
Put these styles in the `body` tag on line 180. Having a duplicate like this can cause confusing behavior, and is also difficult for another person to understand.